ext/pdo_dblib/tests/types.phpt

```diff
@@ -64,3 +69,5 @@ bool(true)
 bool(true)
 bool(true)
 bool(true)
+bool(true)
+bool(true)
```

Comment:
SQLUNIQUE (SYBUNIQUE) is available only since TDS 7.0. =( Maybe I'm wrong, I can't test it with all variants of databases. But when I set tds version 4.6 it really comes as SQLBINARY (45), tds version 6.0+ gives SQLUNIQUE (36). 
Must the extension behave identically at all tds versions? I think 4.* versions should be left off as unsupported by this option. If so I will try to fix the test accordingly.